client/go/internal/cli/cmd/testdata/applications/withEmptyTarget/pom.xml

```diff
@@ -1,2 +1 @@
-<!-- Copyright Vespa.ai. Licensed under the terms of the Apache 2.0 license. See LICENSE in the project root. -->
-<services/>
+<!-- Copyright Vespa.ai. Licensed under the terms of the Apache 2.0 license. See LICENSE in the project root. --> <services/>
```

Comment:
This is a bit silly, but I guess it's new reality with AI coding for us ...

Reply:
Indeed. Fixed with a rebase. 😊